ui/src/app/help/components/help.tsx

```diff
@@ -1,40 +1,78 @@
-                            <p>Want to download the CLI tool?</p>
-                            <a href={`download/argocd-linux-${process.env.HOST_ARCH}`} className='user-info-panel-buttons argo-button argo-button--base'>
-                                <i className='fab fa-linux ' /> Linux
-                            </a>
-                        </div>
-                    </div>
-                    <div className='columns large-4 small-6'>
-                        <div className='help-box'>
-                            <p>You want to develop against Argo CD's API?</p>
-                            <a className='user-info-panel-buttons argo-button argo-button--base' href='/swagger-ui'>
-                                Open the API docs
-                            </a>
-                        </div>
-                    </div>
-                </div>
-            </Page>
-        )}
-    </Consumer>
-);
+export const Help = () => {
+    return (
+        <DataLoader
+            load={() =>
+                combineLatest([services.authService.settings()]).pipe(
+                    map(items => {
+                        return {
+                            binaryUrls: items[0].help.binaryUrls
+                        };
+                    })
+                )
+            }>
+            {({binaryUrls}: {binaryUrls: Record<string, string>}) => {
+                return (
+                    <Consumer>
+                        {() => (
+                            <Page title='Help'>
+                                <div className='row'>
+                                    <div className='columns large-4 small-6'>
+                                        <div className='help-box'>
+                                            <p>New to Argo CD?</p>
+                                            <a className='user-info-panel-buttons argo-button argo-button--base' href='https://argo-cd.readthedocs.io'>
+                                                Read the docs
+                                            </a>
+                                        </div>
+                                    </div>
+                                    <div className='columns large-4 small-6'>
+                                        <div className='help-box'>
+                                            <p>Want to download the CLI tool?</p>
+                                            <a href={`download/argocd-linux-${process.env.HOST_ARCH}`} className='user-info-panel-buttons argo-button argo-button--base'>
+                                                <i className='fab fa-linux' /> Linux
+                                            </a>
+                                            &nbsp;
+                                            {binaryUrls.hasOwnProperty('darwin-amd64') && (
+                                                <a href={`${binaryUrls['darwin-amd64']}`} className='user-info-panel-buttons argo-button argo-button--base'>
```

Comment:
Check whether this works with real HTTP URLs.

Reply:
Yes this works (see bottom left of the screenshot). The download also pops up correctly when I click the link.

<img width="924" alt="image" src="https://user-images.githubusercontent.com/4269898/143097815-eb5e7fff-38f0-4788-b078-2c6d55a53fb6.png">